safe/common/parameters/group_select_parameter_widget.py

```diff
@@ -3,16 +3,17 @@
 
 import logging
 
-from PyQt4.QtCore import Qt
-from PyQt4.QtGui import (
-    QVBoxLayout, QRadioButton, QButtonGroup, QWidget, QLabel, QSizePolicy,
-    QListWidget, QGridLayout, QAbstractItemView, QListWidgetItem)
-
-from parameters.qt_widgets.generic_parameter_widget import (
-    GenericParameterWidget)
-from safe.common.parameters.percentage_parameter_widget import (
-    PercentageSpinBox)
-from safe.definitions.constants import STATIC, SINGLE_DYNAMIC, MULTIPLE_DYNAMIC
+from qgis.PyQt.QtCore import Qt
+from qgis.PyQt.QtWidgets import (QAbstractItemView, QButtonGroup, QGridLayout,
```

Comment:
Pull left -see  https://github.com/inasafe/inasafe/wiki/Coding-Standards#code-formatting

Reply:
@elpaso can you please scan through and fix any others like this too?